src/core-taggeds-optional/Optional/Optional.csproj

```diff
@@ -17,7 +17,7 @@
     <Description>PrimeFuncPack Core.Optional is a core library for .NET consisting of Optional monad targeted for use in functional programming.</Description>
     <RootNamespace>System</RootNamespace>
     <AssemblyName>PrimeFuncPack.Core.Optional</AssemblyName>
-    <Version>2.0.1</Version>
```

Comment:
I suppose adding On-methods needs to increase the minor version (v2.1.0-*) (means new features)

Reply:
Increased to 2.1.0-preview.1